src/napari_imagej/widgets/napari_imagej.py

```diff
@@ -219,22 +231,37 @@ def searchCompleted(self, event: "jc.SearchEvent"):
 
     def _finalize_info_bar(self):
         self.widget.info_box.version_bar.setText(f"ImageJ2 v{ij().getVersion()}")
+        # HACK: Tap into the EventBus to obtain SciJava Module debug info.
+        # See https://github.com/scijava/scijava-common/issues/452
+        event_bus_field = ij().event().getClass().getDeclaredField("eventBus")
+        event_bus_field.setAccessible(True)
+        event_bus = event_bus_field.get(ij().event())
+
+        progress_listener = ProgressBarListener(self.widget.progress_handler)
+        # NB We need to retain a reference to this object or GC will delete it
+        ij().object().addObject(progress_listener)
+        # Subscribe to all ModuleEvents
+        event_bus.subscribe(jc.ModuleEvent.class_, progress_listener)
 
     def _finalize_exception_printer(self):
         # HACK: Tap into the EventBus to obtain SciJava Module debug info.
         # See https://github.com/scijava/scijava-common/issues/452
         event_bus_field = ij().event().getClass().getDeclaredField("eventBus")
         event_bus_field.setAccessible(True)
         event_bus = event_bus_field.get(ij().event())
+
         subscriber = NapariEventSubscriber()
+        # NB We need to retain a reference to this object or GC will delete it
+        ij().object().addObject(subscriber)
         event_bus.subscribe(jc.SciJavaEvent.class_, subscriber)
 
 
 @JImplements(["org.scijava.event.EventSubscriber"], deferred=True)
 class NapariEventSubscriber(object):
     @JOverride
     def onEvent(self, event):
-        log_debug(str(event))
+        pass
+        # log_debug(str(event))
```

Comment:
We might want to file an issue for this, because the printouts can get annoying, but this change should be reverted.

Reply:
It's gone now!

I filed #201 to deal with the annoying printouts.